src/components/MainPage.vue

```diff
@@ -29,6 +28,7 @@
       </v-list-item>
     </v-list>
   </v-container>
+  <BenchmarksInfo :startDate="startDate" :endDate="endDate" />
```

Comment:
While statistics will be important (one day), nobody is using them now. Move them out of sight.

Reply:
Yes, good idea.